domains/blog.crazo7924.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "crazo7924",
+           "email": "",
+           "discord": "466605393309859840",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.WnVEogXILKQlj-jvgWzBYV4EQBNJpb4o31g6aGeiNfpBUziSFOH_A_RYdluWZEm4SN6rwsgh4Ca6BD1FLP-pGkalakxUu22u6oFGPHfclABR0FzYUTwT2ePhQbcCQE0kj_scrL_lUCrWTXfqqu9daPVkKsZSGFR_XtT5stUtwcA_eNuRiRgCB5oteS1Clo3gfWrCIB-XfwUONGfHnm8w_XwiatvVxAV4SMGwh2nNH9lYjLT23rGmO2hd9jptO8GHbNJJWSsndM5D_BIhf1bnDtmuIN3DWciaR9ijmZI5g-CfbkGbph0zhOi4N88eBsdL4Bku5Cu4GnR0_zkjK27cfw.lf05n5TGJ5ekai6Qi7gbdQ.FwSEtzo_z9amY4qpwKLzLct9mtbnkKUEWFSrKlnqv01uRRu1lKQV9_YdU8eZ3UtY0H4wL6iZSrECZPcFxGqGaUl6DIQLc1-jgODQHQNyLzp0WTlMbKcWXCRS80eITxXR.ZaVvxYEKWd03U5gQXIofEw"
+        },
+    
+        "record": {
+            "CNAME": "hashnode.network"
```

Comment:
Can i see a preview of your blog?

Reply:
see the updated first comment of this PR